src/SqlAsyncCollector.cs

```diff
@@ -151,7 +151,7 @@ private async Task UpsertRowsAsync(IEnumerable<T> rows, SqlAttribute attribute,
 
             int batchSize = 1000;
             await connection.OpenAsync();
-            SqlCommand command = connection.CreateCommand();
+            using SqlCommand command = connection.CreateCommand();
```

Comment:
Could you see if there's a rule such as https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2213 we could enable to flag these? Generally we want to make sure we're cleaning up disposables

Reply:
Darn ☹️